static/js/components/google_map.tsx

```diff
@@ -0,0 +1,226 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Component for embedding a map from Google Maps
```

Comment:
please document here that kml coordinates outlines from /mapinfo are preferred over lat/lng pins. that was quite unclear from the outset.

Reply:
Good point! Added more documentation, and also clarified in the inline comments.